docs/src/main/asciidoc/jgit.adoc

```diff
@@ -0,0 +1,54 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - JGit
+
+include::./attributes.adoc[]
+
+
+Quarkus includes the `jgit` extension which enables the use of https://www.eclipse.org/jgit/[Eclipse JGit]
+in native mode.
+
+== Configuration
+
+Once you have your Quarkus project configured you can add the `jgit` extension
+to your project by running the following command in your project base directory.
+
+[source]
+mvn quarkus:add-extension -Dextensions="jgit"
```

Comment:
I would prefer if you added the `----` to delimit the source extract.

Reply:
Good catch. I copied from `kubernetes-client.adoc` :smile: